packages/vite/src/node/plugins/manifest.ts

```diff
@@ -16,6 +17,7 @@ export interface ManifestChunk {
   isDynamicEntry?: boolean
   imports?: string[]
   dynamicImports?: string[]
+  isDuplicate?: true
```

Comment:
Given that we may end up reworking the way duplicates work and removing them I think we should avoid this boolean for the moment.

Reply:
Removed that one. But just wanna ask, when you say "remove them" what are you thinking there?

I know from a back-end perspective we'd always want the duplicate assets to be in the manifest, otherwise you can't reference the file.

Reply:
What I mean is that there is a chance we decide to avoid the dedupe mechanism and let users have the same image twice. In that case, `isDuplicate` wouldn't be needed. So better to wait a bit before adding it.

Reply:
Got ya! It would be cool if that were configurable - but I also know we are too far away to be having those conversations right now.